descartes_utilities/include/descartes_utilities/ros_conversions.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2016, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ *  ros_conversions.h
+ *
+ *  Created on: Feb 28, 2016
+ *  Author: Jonathan Meyer
+ */
+
+#ifndef DESCARTES_ROS_CONVERSIONS_H
+#define DESCARTES_ROS_CONVERSIONS_H
+
+#include <trajectory_msgs/JointTrajectory.h>
+#include <descartes_core/trajectory_pt.h>
+
+namespace descartes_utilities
+{
+  /**
+   * @brief Converts a sequence of Descartes joint trajectory points to ROS trajectory points.
```

Comment:
Regardless, I think this is a bigger issue that needs to be addressed in another pull request. Short of adding velocity and acceleration constraints to the core descartes library, we could create a PR that implements one of the above filters and adds it as a "post-processing filter" to this here `descartes_utilities` package.

Can we accept something akin to this PR and log an issue for post-processing filters?